src/index.css

```diff
@@ -84,7 +84,7 @@ code {
   position: absolute;
   z-index: 1;
   bottom: 42px;
-  left: calc(50% - (3 / 2) * (32px + 2 * 15px));
+  left: calc(50% - (3/2) * (16px + 2*8.5px));
```

Comment:
Let's actually do `calc(50% - (3/2) * (14px + 2*8.5px))` so we use the width of the `<img />` and not the container `<div />`